### ISSUE TITLE
Scrollbar looks ugly and does not autohide itself

### ISSUE THREAD

**Issue by gitSaptarshi** [rafaelmardojai/firefox-gnome-scrollbars#10]
![Screenshot from 2020-08-09 23-24-34](https://user-images.githubusercontent.com/37685073/89738619-bcc27980-da97-11ea-8ca2-b3952017b5d3.png)
As you can see in the screenshot above, the scrollbar inside the hamburger menu looks very ugly and does not autohide itself. I am using the latest build, dated Sun, Aug 9, 2020

### System specification
- OS: Fedora Rawhide
- GNOME version: 3.37.3
- Windowing system: X11
- Firefox version: 78.0.2 (64-bit)

**Comment by gitSaptarshi** [comment]
@rafaelmardojai, please pay a look at this issue. This breaks the aesthetics of the firefox gnome theme.

**Comment by rafaelmardojai** [comment]
In the current implementation, the script only inject CSS. To implement autohide we should write some JS logic.

About the scrollbar position on popovers;
To get a prettier look like this:

![Captura de pantalla de 2020-08-13 12-16-18](https://user-images.githubusercontent.com/6210397/90166421-d695e200-dd5f-11ea-9d1b-2ccaab212c78.png)

We should move the padding from `.panel-arrowcontent` to `.panel-subview-body`, so the scrollbar is drawn on the padding. Also add padding to `.panel-header` & `.panel-footer`. 

The problem is there's a lot of popovers that doesn't use `.panel-subview-body` so we need to add padding to all of them, and each use different classes.

**Comment by gitSaptarshi** [comment]
Yeah I totally got you. Sometimes it's very easy to imagine, but hard to implement things. Keep up this awesome work btw.
